README.md

```diff
@@ -6,7 +6,22 @@
 
 ---
 
-The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. Click the links below for instructions on how to download, install, and use the features of the IntelliJ plugin.
+## Overview
+The Ballerina IntelliJ plugin provides the [Ballerina programming language](https://ballerina.io/) development capabilities within IntelliJ IDEA. While it offers a subset of features compared to the [Ballerina VSCode extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina), it is designed to meet the specific needs of those working within the IntelliJ ecosystem.
+
+> **Note:** This plugin is being developed as an experimental and community-driven project. The plugin is fully open-source, and we welcome contributions!
+
+## Intended Users
+
+This plugin is mainly targeted at **Ballerina library developers** who seek:
```

Comment:
is `library` the correct term here? or should it be package?

Reply:
As Ballerina users usually use packages to structure the projects they are working on, I'm not sure it will give the idea that the plugin is suitable for any package based projects. Here the term `library developers` is used because most of the times they are ones who got to work with both Ballerina and java at the same time. WDYT?